src/components/BuildPanels.vue

```diff
@@ -31,6 +31,53 @@
             {{ panelName }}
           </v-chip>
         </v-card-text>
+
+        <v-card-title>
+          {{ $t('buildPanels.addPanel.text') }}
+        </v-card-title>
+
+        <v-card-text>
+          <v-row>
+            <v-col
+              cols="12"
+              md="8"
+            >
+              <v-file-input
+                v-model="file"
+                label="Upload Panel"
+                prepend-icon="mdi-paperclip"
+              >
+                <template v-slot:selection="{ text }">
+                  <v-chip
+                    small
+                    label
+                    color="primary"
+                  >
+                    {{ text }}
+                  </v-chip>
+                </template>
+              </v-file-input>
+            </v-col>
+            <v-col
+              cols="12"
+              md="4"
+            >
+              <v-tooltip bottom>
+                <template v-slot:activator="{ on }">
+                  <v-btn
```

Comment:
Actually, the button is not needed. The panel should be added immediately when selected instead like with "Update Gene Reference Database". It should also be parsed immediately, without dealing with the other panels. 
Fix the current issue (including removing the button) and we'll deal with revamping the whole tab later.